controllers/user.js

```diff
@@ -61,3 +62,184 @@ exports.create_user = (req, res, next) => {
 
     start();
 };
+
+exports.reset_password = (req, res, next) => {
+    const data = util.get_data(
+        {
+            email: ''
+        },
+        req.body
+    );
+
+    let random_string;
+    
+    function start () {
+        random_string = randomstring.generate();
+
+        if (data instanceof Error) {
```

Comment:
Put this at the beginning of start, para di na masayang yung time gumawa ng random string kung may error pala yung data